backend/.profile

```diff
@@ -11,6 +11,12 @@ export NEW_RELIC_APP_NAME="$(echo "$VCAP_APPLICATION" | jq -r .application_name)
 # Set the environment name for New Relic telemetry.
 export NEW_RELIC_ENVIRONMENT="$(echo "$VCAP_APPLICATION" | jq -r .space_name)"
 
+# Set Agent logging to stdout to be captured by CF Logs
+export NEW_RELIC_LOG=stdout
+
+# Logging level, (critical, error, warning, info and debug). Default to info
+#export NEW_RELIC_LOG_LEVEL=
```

Comment:
I think it is worth while to keep this here, because once we have it working and feeding logs to CF, it may be worth setting this to `error` or `critical` for the future.